### PR TITLE
Set target env only if present

### PR DESCRIPTION
**Pull request by olemoi**
Check if target environment is present before adjusting RJ value

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -734,6 +734,7 @@ k8s.io/apimachinery v0.0.0-20190809020650-423f5d784010/go.mod h1:Waf/xTS2FGRrgXC
 k8s.io/apimachinery v0.0.0-20191016225534-b1267f8c42b4/go.mod h1:92mWDd8Ji2sw2157KIgino5wCxffA8KSvhW2oY4ypdw=
 k8s.io/apimachinery v0.0.0-20191020214737-6c8691705fc5 h1:r3/YL3+t1U46lJF5zUSArskUpnLyWuM28rQDpM1qQPI=
 k8s.io/apimachinery v0.0.0-20191020214737-6c8691705fc5/go.mod h1:92mWDd8Ji2sw2157KIgino5wCxffA8KSvhW2oY4ypdw=
+k8s.io/apimachinery v0.18.6 h1:RtFHnfGNfd1N0LeSrKCUznz5xtUP1elRGvHJbL3Ntag=
```
Comment (satr) [inline go.sum:737]: Is this change needed?

Comment (olemoi) [reply go.sum:737]: To my understanding, go.sum changes are nice to have in case someone has to make sure they're using the same modules.
